spec/robots/dor_repo/gis_assembly/generate_content_metadata_spec.rb

```diff
@@ -29,8 +29,7 @@
 
     before do
       allow(Dor::Services::Client).to receive(:object).and_return(object_client)
-      allow(SecureRandom).to receive(:uuid).and_return('8222376b-861f-4cb1-8ebb-c2ae6b112b4e', 'ec13ab89-39b0-455d-8b2c-f6e1c9cc8e60', '0d896cd1-57e3-4fc1-93c0-dd0e37d4e65a',
-                                                       'd665cf1c-9914-45bc-82e7-bbf946a614d8')
+      allow(SecureRandom).to receive(:uuid).and_return('8222376b-861f-4cb1-8ebb-c2ae6b112b4e', 'ec13ab89-39b0-455d-8b2c-f6e1c9cc8e60', '0d896cd1-57e3-4fc1-93c0-dd0e37d4e65a')
```

Comment:
one less file getting generated  == one less UUID to mock

Reply:
which also means, the ones we expect all shift up a slot, updated in specs below